VideoLocker/res/values/dimens.xml

```diff
@@ -70,6 +70,7 @@
     <dimen name="horizontal_input_padding">@dimen/widget_margin</dimen>
     <dimen name="vertical_input_padding">8dp</dimen>
     <dimen name="edx_button_height">40dp</dimen>
+    <dimen name="create_new_item_icon_margin">3dp</dimen>
```

Comment:
Its in the correct section, the section has elements not having the `discussion` prefix in their name and are being mostly used in discussion UIs.